packages/static-hosting/lib/handlers/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
remove this if no scripst are setup?

Reply:
I just copied that from the other handlers :sweat_smile: I can remove it though